internal/server/handlers/updatemodels.go

```diff
@@ -0,0 +1,72 @@
+package handlers
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/npavlov/go-metrics-service/internal/domain"
+	"github.com/npavlov/go-metrics-service/internal/server/db"
+)
+
+func (mh *MetricHandler) UpdateModels(response http.ResponseWriter, request *http.Request) {
+	ctx, cancel := context.WithTimeout(request.Context(), mh.timeout)
```

Comment:
[CHANGE REQUEST] This `context` should already be with timeout.

Reply:
added middleware with timeout